### DISCUSSION TITLE
renaming the default branch to main

### DISCUSSION THREAD

**Discussion by gireeshpunathil** [eclipse-equinox/equinox#299] [General]
rationale: align with several world-wide initiatives on avoiding oppressive metaphors in open source projects

github has already made main as the default for new repositories.

I would like to propose the default branch name change in eclipse SDK projects. In a similar request[1] to foundation earlier, the response was that they don't want to enforce this change, while supporting it.

I have started a discussion thread in PDE[2]. The observation is that major focus would be in the releng-aggregator repo, while for other repos it is mostly the name switch plus changing any hard-coded references in the jenkins job. Luckily the scripts in the releng-aggregator is already capable of configuring any branch names with relative ease.

I am not a committer in this project, so cannot drive this action to completion. Can I request:
 - opinions from project members on this matter
 - one or more committers who would like to anchor this for making progress?

thanks in advance!

[1] https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/535
[2] https://github.com/eclipse-pde/eclipse.pde/discussions/657

**Comment by iloveeclipse** [comment]
I've [already commented on platform issue](https://github.com/eclipse-pde/eclipse.pde/discussions/657#discussioncomment-6377053).
